### PR TITLE
add glossary

### PR DESCRIPTION
**Pull request by Lilyhanig**
- this could potentially be done differently I think, with a mustache and json file like the docstrings in PowerSystems/InfrastructureSystems.... maybe more work than it's worth.

### CODE REVIEW

File: src/core/glossary.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Glossary
+
+## Simulation Sequence Components
+
+**cache:** Dictionary of each stage and its vector of caches. Each cache tracks the last time status of a device changed in a simulation.
```
Comment (jd-lara) [inline src/core/glossary.md]: Don't mention the type of object, just the definition for instance in this case: 

`Cache is the storage of the solution of a variable from a stage to be used by other stages in the simulation`

File: src/core/glossary.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**feed_forward:** The vertical inter-stage relationship between the binary variable status of one stage and the affected variables of the next stage.
+- Example: `feedforward = Dict(("stage-2", :devices, :Generators) => SemiContinuousFF(binary_from_stage = PSI.ON, affected_variables = [PSI.ACTIVE_POWER])`
+
+**feed_forward_chronologies:** The vertical inter-stage relationship dictating how variable results impact the next stage's variable parameters.
```
Comment (jd-lara) [inline src/core/glossary.md]: mention that this creates a time relationship.

File: src/core/glossary.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Glossary
+
+## Simulation Sequence Components
+
+**cache:** Dictionary of each stage and its vector of caches. Each cache tracks the last time status of a device changed in a simulation.
+
+**feed_forward:** The vertical inter-stage relationship between the binary variable status of one stage and the affected variables of the next stage.
```
Comment (jd-lara) [inline src/core/glossary.md]: Not a relationship, it is the variable that is used as a parameter for a later stage.

File: src/core/glossary.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Example: `feedforward = Dict(("stage-2", :devices, :Generators) => SemiContinuousFF(binary_from_stage = PSI.ON, affected_variables = [PSI.ACTIVE_POWER])`
+
+**feed_forward_chronologies:** The vertical inter-stage relationship dictating how variable results impact the next stage's variable parameters.
+- Example: `feedforward_chronologies = Dict(("stage-1" => "stage-2") => Synchronize(periods = 24))`
+
+**horizons:** The integer count of resolution time periods for a full step resolution of the simulation. *(Horizon = 12) x (resolution = 1 Hour) = 12 Hours*
```
Comment (jd-lara) [inline src/core/glossary.md]: Horizon

File: src/core/glossary.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- Example: `horizons = Dict("stage-1" => 24, "stage-2" => 12)`
+
+**ini_cond_chronology:** The structure dictating how initial conditions get updated from previous results in the simulation.
```
Comment (jd-lara) [inline src/core/glossary.md]: don't use the input to the simulation object. use just the words. 

Initial Condition Chronology

File: src/core/glossary.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+- Example: `ini_cond_chronology = InterStageChronology()`
+
+**intervals:** The increment of time per stage in each simulation solve, and how results get fed forward into initial conditions between intervals.
```
Comment (jd-lara) [inline src/core/glossary.md]: Interval*

File: src/core/glossary.md (hunk 3 above)
Comment (daniel-thom) [reply src/core/glossary.md]: Jose recently changed all uses to `feedforward`

**Pull request closed without merging**